tools/sizes.py

```diff
@@ -23,6 +23,15 @@
 import subprocess
 import sys
 
+def get_segment_hints():
+    hints = {}
+    hints['IROM'] = '      code in flash (default or ICACHE_FLASH_ATTR)'
+    hints['IRAM'] = '/32KB code in IRAM (ICACHE_RAM_ATTR, ISRs...)'
```

Comment:
The LHS is in raw bytes, so I suggest the RHS should also be in bytes ("28444 out of 32768" is easier to understand for folks who don't know what 32*1024 is)